Entities/Items/Crate/Crate.as

```diff
@@ -162,6 +185,36 @@ bool doesCollideWithBlob(CBlob@ this, CBlob@ blob)
 	return !blob.hasTag("parachute");
 }
 
+void onCollision( CBlob@ this, CBlob@ blob, bool solid ){
+	
+	if(!isServer())return;
+	
+	if(blob !is null){
+		if (!blob.isAttached() && blob.isOnGround() && blob.hasTag("material") && blob.getName() != "mat_arrows")
```

Comment:
Short-circuiting makes it possible to merge the two conditions: `blob !is null && !blob.isAttached() && ...`

Reply:
Got it!